### PR TITLE
Added text changes as requested from Hayden in admissions slack channel

### PR DESCRIPTION
**Pull request by bspeare**
Verify code changes reflect:

Change "Cambridge International Exams" to "General Certificate of Education (GCE)"

### CODE REVIEW

File: config/sites/admissions.uiowa.edu/views.view.areas_of_study.yml [hunk 1]
```diff
@@ -1097,7 +1097,7 @@ display:
...
...\n    <tr>\r\n      <td><a href=\"/print/pdf/node/461\" title=\"General Certificate of Education (GCE)\">General Certificate of Education (GCE)</a></td>\r\n      <td><a href=\"/node/461/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/451\" title=\"College Level Examination Program (CLEP)\">College Level Examination Program (CLEP)</a></td>\r\n      <td><a href=\"/node/451/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/456\" title=\"International Baccalaureate Program (IB)\">International Baccalaureate Program (IB)</a></td>\r\n      <td><a href=\"/node/456/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/446\">Advanced Placement Program (AP)</a></td>\r\n      <td><a href=\"/node/446/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n    <tr>\r\n      <td><a href=\"/print/pdf/node/4211\">Global Assessment Certificate Program (GAC)</a></td>\r\n      <td><a href=\"/node/4211/edit\" media_library=\"Media Library\">edit</a></td>\r\n    </tr>\r\n  </tbody>\r\n</table>"
```
Comment (pyrello) [inline config/sites/admissions.uiowa.edu/views.view.areas_of_study.yml]: Not totally clear on what all is supposed to be in here but this looks wrong: `<td><a href=\"/node/456/edit\" media_library=\"Media Library\">edit</a></td>\r\n`

Comment (bspeare) [reply config/sites/admissions.uiowa.edu/views.view.areas_of_study.yml]: Yes, that is pretty bizarre to see!

**Review by cory-skeers — APPROVED** [review]
Looks good to go!

**Comment by bspeare** [comment]
Thanks @cory-skeers !

**Pull request merged**